adal/tests/Test.ADAL.NET.UIAutomation/XamarinDroidADALTests.cs

```diff
@@ -79,7 +79,7 @@ public void AcquireTokenInteractiveTest()
         [Test]
         public void AcquireTokenSilentTest()
         {
-            _adalMobileTestHelper.AcquireTokenSilentTestHelper(xamarinController, LabUserHelper.DefaultUserQuery);
+            _adalMobileTestHelper.AcquireTokenSilentTestHelper(xamarinController, LabUserHelper.GetUser(LabUserHelper.DefaultUserQuery));
```

Comment:
>LabUserHelper.GetUser [](start = 82, length = 21)

Consider adding a LabUserHelper.GetDefaultUser, this construct seems a bit convoluted and it does many things. 
Note: given that it's likely more returning a labResponse, then the mehtod name should probably be slightly different. 
E.g.: GetLabResponseWithDefaultUser